znncli_general.go

```diff
@@ -0,0 +1,234 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/hypercore-one/go-zdk/utils"
+	"github.com/hypercore-one/go-zdk/utils/template"
+	"github.com/urfave/cli/v2"
+	"github.com/zenon-network/go-zenon/common/types"
+)
+
+// TODO message data
+var znnCliSend = &cli.Command{
+	Name:  "send",
+	Usage: "toAddress amount zts",
+	Action: func(cCtx *cli.Context) error {
+		if cCtx.NArg() != 3 {
+			fmt.Println("Incorrect number of arguments. Expected:")
+			fmt.Println("send toAddress amount zts")
+			return nil
+		}
+
+		kp, err := getZnnCliSigner(walletDir, cCtx)
+		if err != nil {
+			fmt.Println("Error getting signer:", err)
+			return err
+		}
+		z, err := connect(url, chainId)
+		if err != nil {
+			fmt.Println("Error connecting to Zenon Network:", err)
+			return err
+		}
+
+		toAddress, err := types.ParseAddress(cCtx.Args().Get(0))
+		if err != nil {
+			fmt.Println("Error bad toAddress:", err)
+			return err
+		}
+
+		zts, err := getTokenStandard(cCtx.Args().Get(2))
+		if err != nil {
+			fmt.Println("Error bad zts:", err)
+			return err
+		}
+
+		token, err := z.Embedded.Token.GetByZts(zts)
+		if err != nil {
+			fmt.Println("Error fetching zts:", err)
+			return err
+		}
+
+		// TODO handle decimal amounts
+		amount := big.NewInt(0)
+		amount, ok := amount.SetString(cCtx.Args().Get(1), 10)
+		if !ok {
+			fmt.Println("Error bad amount")
+			return nil
+		}
+
+		amount = amount.Mul(amount, big.NewInt(10^int64(token.Decimals)))
```

Comment:
The `^`-symbol isn't actually raising to a power of and will result in an incorrect amount being sent.
```suggestion
		amount = amount.Mul(amount, new(big.Int).Exp(big.NewInt(10), big.NewInt(int64(token.Decimals)), nil))
```